models/agent_model.go

```diff
@@ -170,6 +171,9 @@ func (s *Agent) BeforeInsert() error {
 	if len(s.CustomLabels) == 0 {
 		s.CustomLabels = nil
 	}
+	if s.Status == "" && s.AgentType != ExternalExporterType && s.AgentType != PMMAgentType {
+		s.Status = inventorypb.AgentStatus_name[int32(inventorypb.AgentStatus_UNKNOWN)]
```

Comment:
can we have a separate agent status enum on the model layer? e.g. like we have for NodeType 
https://github.com/percona/pmm-managed/blob/PMM-2.0/models/node_model.go#L33

Reply:
I added a constant for UNKNOWN as we don't need other statuses.